Example/MTCircularSlider/ViewController.swift

```diff
@@ -53,7 +53,7 @@ class ViewController: UIViewController {
 	}
 
 	fileprivate func setValueLabelText() {
-		valueLabel.text = String(Int(knobWithLabelView.value))
+		valueLabel.text = String(Int(knobWithLabelView.value))        
```

Comment:
Why are we deleting the round caps from the example?

Reply:
I put that one, before didn't have that property, just to be sure for the rounds cap its ok with the version 12.0 its working

Reply:
Let's revert the spacing change here and we're good to go 🙂

Reply:
ok!